### ISSUE TITLE
Error in message formatter config

### ISSUE THREAD

**Issue by vikumkbv** [wso2/docs-apim#7556]
Hi Team,

In the documentation [1], we found a typo in the [[custom_message_formatters]] class, please correct is as follows.

**Current version**

```
[[custom_message_formatters]]
class = "org.wso2.carbon.relay.BinaryRelayBuilder"
content_type = "application/json/badgerfish"

[[custom_message_builders]]
class = "org.wso2.carbon.relay.BinaryRelayBuilder"
content_type = "application/json/badgerfish"
```

**Corrected version**

```
[[custom_message_formatters]]
class = "org.wso2.carbon.relay.ExpandingMessageFormatter"
content_type = "application/json/badgerfish"

[[custom_message_builders]]
class = "org.wso2.carbon.relay.BinaryRelayBuilder"
content_type = "application/json/badgerfish"
```

[1] https://apim.docs.wso2.com/en/latest/install-and-setup/setup/mi-setup/message_builders_formatters/message-builders-and-formatters/#handling-message-relay

Thanks,
Vikum

**Comment by DinithiDiaz** [comment]
> @DinithiDiaz , I have fixed this issue in [1]. Please review and merge the PR.

Hi @Mariangela, merged the [PR](https://github.com/wso2/docs-apim/pull/7558).

**Comment by DinithiDiaz** [comment]
Sent fixes via https://github.com/wso2/docs-mi/pull/69 and https://github.com/wso2/docs-mi/pull/70 to update the [MI 4.2.0 documentation](https://mi.docs.wso2.com/en/latest/install-and-setup/setup/message-builders-formatters/message-builders-and-formatters/).

**Comment by Mariangela** [comment]
@DinithiDiaz , I am assuming that this fix is valid for 4.1.0 and 4.0.0 as well. Therefore, I made these changes in the master, 4.1.0 and 4.0.0 branches.
